types.go

```diff
@@ -10,18 +10,19 @@ import (
 
 // LinodeType represents a linode type object
 type LinodeType struct {
-	ID         string          `json:"id"`
-	Disk       int             `json:"disk"`
-	Class      LinodeTypeClass `json:"class"` // enum: nanode, standard, highmem, dedicated, gpu
-	Price      *LinodePrice    `json:"price"`
-	Label      string          `json:"label"`
-	Addons     *LinodeAddons   `json:"addons"`
-	NetworkOut int             `json:"network_out"`
-	Memory     int             `json:"memory"`
-	Transfer   int             `json:"transfer"`
-	VCPUs      int             `json:"vcpus"`
-	GPUs       int             `json:"gpus"`
-	Successor  string          `json:"successor"`
+	ID           string              `json:"id"`
+	Disk         int                 `json:"disk"`
+	Class        LinodeTypeClass     `json:"class"` // enum: nanode, standard, highmem, dedicated, gpu
+	Price        *LinodePrice        `json:"price"`
```

Comment:
Just curious, since we have region-specific pricing, is `Price` going to be deprecated?

Reply:
Good question! As of now I think the `region_prices` field will only contain exceptions and the price field will dictate the price for all other regions.